### PR TITLE
Jenkins Docker Image

### PR DESCRIPTION
**Pull request by rajeevr2715**

### CODE REVIEW

**Comment by rajeevr2715** [comment]
@suryajak Can you please review. Seems a permission issue in the repo.

**Comment by bleggett** [comment]
See https://github.com/tmobile/jazz-installer/pull/332#pullrequestreview-151036984 before merging.

**Comment by devsatishm** [comment]
I don't think that jazz-content is the correct location for these artifacts. We're using jazz-content for pure static content (like image/video etc.).

**Comment by suryajak** [comment]
The original intent for jazz-content was to use it to store large file sizes. We talked about moving fairly static content over there which would also include build dependencies and hence I had created directories like `jenkins` and `codeq` as placeholders. 

Even while the Jenkins docker file doesn't fit into that definition entirely, we had the option of creating another repo, or repurpose `jazz-content` repo for it. The rationale behind choosing the latter was that the contents that go into building the image are fairly static and so is the  image building process.

**Comment by bleggett** [comment]
@devsatishm Right, we are storing the image in Dockerhub. But the static artifacts we use to _build_ the image in our CI, _those_ need to go in a repo.

Putting them in a separate repo is A) best practice and B) Lets us trigger the image build and the push of a new version to Dockerhub when changes are made to the static artifacts we use to build the container.

We want to trigger the docker image build and the push to Dockerhub separate from the installer build, they do not need to be in the same repo.

File: jenkins/plugins.txt [hunk 1]
```diff
@@ -0,0 +1,85 @@
+bouncycastle-api:2.16.1
```
Comment (devsatishm) [inline jenkins/plugins.txt]: Most of these plugins require updates. But, I guess we are covering this as part of a separate work item.

Comment (rajeevr2715) [reply jenkins/plugins.txt]: Yes. We will handle it in jenkins security related story.

**Pull request merged**